### ISSUE TITLE
Use `http` in ORCIDs as they are entered in the editors

### ISSUE THREAD

**Issue by laurenwalker** [NCEAS/metacatui#1368]
In the ORCID text inputs in the access policy editor or metadata editor, we should convert ORCID strings with `https` to `http`, since that is the format used by the DataONE auth portal when DataONE accounts are created.

**Comment by mbjones** [comment]
Such a bummer, given that we have recommended https for all access policies with ORCIDs. It seems to me that the auth portal really needs to treat http and https as equivalent for ORCIDs.  I think we had a ticket discussing these issues somewhere...

**Comment by amoeba** [comment]
yep, @mbjones , we got stuck on this over on https://redmine.dataone.org/issues/8215.

**Comment by laijasmine** [comment]
bumping this issue again to automatically convert ORCiDs entered from `https` to `http`. This issue will come up more as more portals get created and shared so prioritizing this fix is needed.

**Comment by laurenwalker** [comment]
Thanks Jasmine, I thought we were doing this already, but if not, I'll get that fix in.

**Comment by laijasmine** [comment]
Thanks Lauren!

I just had a researcher write in and they were able to add users with ORCiD starting with `https`. I found that the permissions do not automatically convert ORCiDs to `http` when I tested it on test.arcticdata.io
